### ISSUE TITLE
Dragging modules doesn't work correctly with Inner Rows

### ISSUE THREAD

**Issue by SlavaET** [elegantthemes/Divi-Beta#352]
### Problem:

I cannot move the modules into Inner Rows ( which are in the Specialty sections ) or move modules outside of these rows. I see weird behaviour and it doesn't work as expected.

I can copy/paste with right click menu without any issues though. Looks like something wrong with sortable logic.

![specialty-drag](https://cloud.githubusercontent.com/assets/3446682/18264601/09891578-7433-11e6-9202-08480dc25ba8.gif)

### Steps To Reproduce:
1. Try moving module into the Inner Row using Visual Builder
2. Try moving module from the Inner Row using Visual Builder

**Comment by SlavaET** [comment]
@fikrirasyid I still see some issues there:
- [ ] If I move the regular Row into Specialty Section then it breaks the section. Probably we shouldn't allow the moving Regular Rows into Specialty Rows as in BB:
![inner-row-errors](https://cloud.githubusercontent.com/assets/3446682/18272142/030a2424-7458-11e6-8ba4-cb0134cbbcb6.gif)

- [ ] It's still possible to drop a module into wrong position and module will disappear:
![inner-row-errors-2](https://cloud.githubusercontent.com/assets/3446682/18272231/8bed5f54-7458-11e6-9232-6cd42f2e1ac8.gif)

**Comment by timayhelen** [comment]
@fikrirasyid In this branch it's possible to move normal rows inside of specialty sections. This shouldn't be allows, and it also throws a lot of errors:

![screen shot 2016-09-06 at 9 16 46 am](https://cloud.githubusercontent.com/assets/3333028/18281687/a89c944e-7412-11e6-8913-b66eafe460c3.png)

**Comment by fikrirasyid** [comment]
@SlavaET @timayhelen I've pushed https://github.com/elegantthemes/submodule-builder/commit/5b8bb61868920d62c94f6aeff01b992779b5aa3b which should avoid regular to be moved to specialty section's row. 

***

I'm still working on the second one:

> It's still possible to drop a module into wrong position and module will disappear:

I've noticed that the module is actually doesn't disappear, but its module's inline styling makes the module invisible. I haven't found a good way removed those irrelevant inline styling tho. I'm still working on it.
